lib/game.rb

```diff
@@ -0,0 +1,99 @@
+require 'pry'
+
+class Game
+
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  WIN_COMBINATIONS = [
+    [0, 1, 2],#top
+    [3, 4, 5],#mid
+    [6, 7, 8],#bot
+    [0, 3, 6],#left vert
+    [1, 4, 7],#mid vert
+    [2, 5, 8],#right vert
+    [0, 4, 8],#slash1
+    [2, 4, 6]#slash2
+  ]
+
+  def current_player
+    @board.turn_count % 2 == 0 ? @player_1 : @player_2
+  end
+
+  def over?
+    won? || draw?
+  end
+
+  def won?
+    WIN_COMBINATIONS.detect do |combo|
+      @board.cells[combo[0]] == @board.cells[combo[1]] &&
+      @board.cells[combo[1]] == @board.cells[combo[2]] &&
+      @board.taken?(combo[0] + 1)
+    end
+  end
+
+  def draw?
+    @board.full? && !won?
+  end
+  
+  def winner
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.